docs/getting_started.md

```diff
@@ -1,8 +1,8 @@
-## Getting started with Mbed Crypto
+## Getting started with Mbed TLS
 
-### What is Mbed Crypto?
+### What is Mbed TLS?
 
-Mbed Crypto is an open source cryptographic library that supports a wide range of cryptographic operations, including:
```

Comment:
Historically, Mbed Crypto was the cryptographic part of Mbed TLS, so all of these descriptions will only partially be true. Mbed TLS also has x509 and TLS APIs, so describing it as only the cryptographic part is inaccurrate.